src/main/java/seedu/address/logic/commands/CommandResult.java

```diff
@@ -19,21 +19,26 @@ public class CommandResult {
     /** The application should exit. */
     private final boolean exit;
 
+    /** Group UI should be shown. */
+    private final boolean groupCommand;
+
     /**
      * Constructs a {@code CommandResult} with the specified fields.
      */
-    public CommandResult(String feedbackToUser, boolean showHelp, boolean exit) {
+    public CommandResult(String feedbackToUser, boolean showHelp,
```

Comment:
Probably have to resolve merge conflicts in this class because previous commit added an additional field to this constructor for the clear command